examples/sdk/basic-javascript-example/README.md

```diff
@@ -0,0 +1,11 @@
+# Basic Vanilla Javascript SDK example
+
+In this example, the SDK gets initialised in the most simple form.
+It loads in the SDK without any packagemanager, just as a script from our CDN endpoint, using always the latest version in the process.
```

Comment:
To load something in is correct English
"To put or insert something into something else, such as an appliance, structure, vehicle, etc."

Since you load the SDK into your own application I feel this can stay? Both are probably correct 😄